src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -215,13 +215,13 @@ private String getSubmissionFormTableHeaderFragmentHtml(String questionNumberStr
         StringBuilder tableHeaderFragmentHtml = new StringBuilder();
         String tableHeaderFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_SUBMISSION_FORM_HEADER_FRAGMENT;
 
-        for(int i = 0 ; i < numOfRubricChoices ; i++) {
+        for(int i = 0; i < numOfRubricChoices; i++) {
             String tableHeaderCell = 
                     FeedbackQuestionFormTemplates.populateTemplate(tableHeaderFragmentTemplate,
                             "${qnIndex}", questionNumberString,
                             "${respIndex}", responseNumberString,
                             "${col}", Integer.toString(i),
-                            "${rubricChoiceValue}", rubricChoices.get(i));
+                            "${rubricChoiceValue}", (rubricChoices.get(i) + " (" + (numOfRubricChoices-i) + ")"));
```

Comment:
`numOfRubricChoices - i` don't forget the whitespace before/after the minus sign. Similarly for other arithmetic operators (+ * /).